src/stores/config.store.ts

```diff
@@ -39,6 +40,7 @@ export const useThemeStore = defineStore(
 
     function setTheme(name: string) {
       themeName.value = name
+      themeSelectorService.setCurrentThemeColors(name)
```

Comment:
Usually we would handle the logic inside a component instead of doing this inside of the store.
Eg. add a watcher on theme name in `theme-selector.vue` and call `themeSelectorService.setCurrentThemeColors(name)` in the watcher.
But it is working great as is too, so you can make the change if you find it necessary, or leave it as is, it's up to you.